vega-lite-schema.json

```diff
@@ -1025,6 +1025,12 @@
             },
             "type": "object"
         },
+        "Dict<SelectionSpec>": {
```

Comment:
I defer to whatever you two think is the right thing to do! This is what we'd used in the InfoVis prototype, and I just copied it over for expediency's sake since I'm still pretty unfamiliar with TypeScript.